tests/appsec/test_blocking_addresses.py

```diff
@@ -584,11 +589,11 @@ def test_request_block_attack(self):
                 parameters["address"] == "graphql.server.all_resolvers"
                 or parameters["address"] == "graphql.server.resolver"
             )
-            assert rule_triggered["rule"]["id"] == (
-                "block-resolvers" if parameters["address"] == "graphql.server.resolver" else "block-all-resolvers"
-            )
+            assert rule_triggered["rule"]["id"] == "block-resolvers"
             assert parameters["key_path"] == (
-                ["userByName", "name"]
+                ["getUserByName", "0", "name"]
```

Comment:
I'd add a comment clarifying why the key_path is different so next implementaters know about this. In order to solve this on the long run and get consistent results we might discuss about modifying the RFC to redefine the WAF's payload format. Other than that LGTM.